go/consensus/tendermint/seed.go

```diff
@@ -168,7 +171,10 @@ func populateAddrBookFromGenesis(addrBook p2p.AddrBook, doc *genesis.Document, o
 		var tmvAddr *p2p.NetAddress
 		tmvAddr, err := api.NodeToP2PAddr(&openedNode)
 		if err != nil {
-			return errors.Wrap(err, "tendermint/seed: failed to reformat genesis validator address")
+			logger.Error("tendermint/seed: failed to reformat genesis validator address",
```

Comment:
No need to include the `tendermint/seed` prefix, instead use `consensus/tendermint/seed` for the logger module. Also below.

Reply:
ok